app/assets/stylesheets/components/placeholder.scss

```
@@ -48,7 +48,7 @@
   top: 0.1em;
 
   .sms-message-wrapper & {
-    box-shadow: inset 0 -0.35em 0 0 govuk-shade(govuk-colour("light-grey"), 7%);
+    box-shadow: inset 0 -0.25em 0 0 govuk-shade(govuk-colour("light-grey"), 7%);
```

This same problem was fixed for emails in https://github.com/alphagov/notifications-admin/pull/4438

I didn’t think to check if there was different styling for hidden placeholders in text messages. Nice work screenshotting all the things.

This very slight tweak will make it match emails exactly, where we added a second inner shadow to get the vertical alignment of the black box with with the text looking balanced:
```suggestion
    box-shadow: inset 0 0.05em 0 0 govuk-shade(govuk-colour("light-grey"), 7%), inset 0 -0.25em 0 0 govuk-shade(govuk-colour("light-grey"), 7%);
```